modules/sdk-unified-wallet/src/ecdsa-unified-wallet/types.ts

```diff
@@ -1,4 +1,5 @@
-import { CoinFamily } from '@bitgo/statics';
+import { Gteth, Eth } from '@bitgo/sdk-coin-eth';
+import { Tpolygon, Polygon } from '@bitgo/sdk-coin-polygon';
 
-export const supportedCoins: string[] = [CoinFamily.ETH, CoinFamily.POLYGON];
-export type EvmWalletCoins = typeof supportedCoins[number];
+export const supportedCoins: string[] = [Eth.name, Polygon.name].map((e) => e.toLowerCase());
+export const supportedTestCoins: string[] = [Gteth.name.toLowerCase(), Tpolygon.name].map((e) => e.toLowerCase());
```

Comment:
this module cannot depend on other coin modules outside of devDependencies. the list of supported coin names just comes from statics anyways, please use that instead.